langlib/langlib-test/src/test/resources/test-src/regexp_test.bal

```diff
@@ -61,6 +61,61 @@ function testFind() {
     assertEquality(6, resultSpan6.startIndex);
     assertEquality(12, resultSpan6.endIndex);
     assertEquality("Botter", resultSpan6.substring());
+
+    var regExpr6 = re `(?im-sx:H*ere?!?)`;
+    regexp:Span? res6  = regExpr6.find("match here", 5);
+    assertTrue(res6 is regexp:Span);
+    regexp:Span resultSpan7 = <regexp:Span>res6;
+    assertEquality(6, resultSpan7.startIndex);
+    assertEquality(10, resultSpan7.endIndex);
+    assertEquality("here", resultSpan7.substring());
+    
+    var regExpr7 = re `${"3*"}`;
+    regexp:Span? res71  = regExpr7.find("1233333333", 5);
+    assertTrue(res71 is regexp:Span);
+    regexp:Span resultSpan81 = <regexp:Span>res71;
+    assertEquality(5, resultSpan81.startIndex);
+    assertEquality(10, resultSpan81.endIndex);
+    assertEquality("33333", resultSpan81.substring());
+
+   //  regexp:Span? res72  = regExpr7.find("");
```

Comment:
tag the relevant PR in the commented sections

Reply:
Will do